TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Section.js

```diff
@@ -44,7 +47,7 @@ define(
 			if (!this.get('inspector.configuration.' + this.get('_nodeType'))) {
 				this.set('inspector.configuration.' + this.get('_nodeType'), {});
 			}
-			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('group'), this.get('_collapsed'));
+			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('groupName'), this.get('_collapsed'));
```

Comment:
why are you suddenly using 'groupName' here? And this.get('group.collapsed') before? Sounds like eager boyscouting at first sight ;-)

Regarding the solution: I'd be fine with it, but that's a vote without knowing askes proposed solution ;-)